svgscreen/svgscreen.go

```diff
@@ -235,12 +257,33 @@ func (s *Screen) setupBgRects() {
 	}
 }
 
+func setupCustomColors(revLookup map[string]int, clsTable *[]string) {
+	result := make([]string, len(revLookup))
+	for k, v := range revLookup {
+		result[v] = k
+	}
+	*clsTable = result
+}
+
 func (s *Screen) Render(w io.Writer) error {
 	t := template.New("")
 	t.Funcs(template.FuncMap{
 		"base64": func(bs []byte) string { return base64.RawStdEncoding.EncodeToString(bs) },
+		"anyTrue": func(arr [16]bool) bool {
```

Comment:
Could this be `arr []bool`? btw i usually use the convention that slice variables names end with `s`, ex `bs` for byte slice

... now i see `ANSIUsed`  is `[16]bool`. If you want it to be a slice you have to assign `ANSIUsed` using `make([]bool, 16)` etc.

Maybe what mostly confused me was the name. Keep type and more explicit name like `anyColorTrue` etc?

Reply:
to keep it simple I'll just change the name

Reply:
👍